From cdb8423130d470a56402e0abc46343658f7ac5ed Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Thu, 23 Jul 2015 17:04:57 +0800
Subject: [PATCH] dts:arm:rk3288-box: ddr frequency set to 533 when playing 4K
 video.

If we playing 4K60Hz video, and there are more than two ui layer which
is continuously refreshing, bandwith will be not enougth under ddr 456MHz.

Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-box.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3288-box.dts b/arch/arm/boot/dts/rk3288-box.dts
index d20ff68ea994..a2dcadd308f3 100644
--- a/arch/arm/boot/dts/rk3288-box.dts
+++ b/arch/arm/boot/dts/rk3288-box.dts
@@ -628,7 +628,7 @@
 		SYS_STATUS_NORMAL	456000
 		SYS_STATUS_SUSPEND	200000
 		//SYS_STATUS_VIDEO_1080P  240000
-		SYS_STATUS_VIDEO_4K     456000
+		SYS_STATUS_VIDEO_4K     533000
 		SYS_STATUS_PERFORMANCE  533000
 		//SYS_STATUS_BOOST	324000
 		//SYS_STATUS_ISP	400000
-- 
2.35.3

